custom/enikshay/private_sector_datamigration/models.py

```diff
@@ -0,0 +1,490 @@
+    phoneNumber = models.CharField(max_length=255, null=True)
+    retreatmentReason = models.CharField(max_length=255, null=True)
+    rxArchivalDate = models.DateTimeField(null=True)
+    rxAssignedBy = models.CharField(max_length=255, null=True)
+    rxInitiationStatus = models.CharField(max_length=255, null=True)
+    # [u'Died before treatment start',
+    #  u'Lost to follow up/ unable to capture',
+    #  u'On treatment (Program support)',
+    #  u'On treatment (public)',
+    #  u'On treatment (self)',
+    #  u'Patient missing',
+    #  u'Pending']
+    rxOutcomeDate = models.DateTimeField(null=True)
+    rxStartDate = models.DateTimeField(null=True)
+    rxSupervisor = models.CharField(max_length=255, null=True)
+    site = models.CharField(max_length=255, null=True)
+    status = models.CharField(max_length=255, null=True)
+    treatingQP = models.CharField(max_length=255, null=True)
+    treatmentOutcomeId = models.CharField(max_length=255, null=True)
+    treatmentPhase = models.CharField(max_length=255, null=True)
+    # [u'Select',
+    #  None,
+    #  u'Intensive Phase',
+    #  u'N/A',
+    #  u'Continuation Phase',
+    #  u'Extended IP']
+    tsProviderType = models.CharField(max_length=255, null=True)
+    unknownAdherencePct = models.DecimalField(decimal_places=10, max_digits=14)
+    unresolvedMissedDosesPct = models.DecimalField(decimal_places=10, max_digits=14)
+    treatingHospital = models.CharField(max_length=10, null=True)
+    treatmentCompletionInsentiveFlag = models.CharField(max_length=1, null=True)
+    mermIMIEno = models.CharField(max_length=255, null=True)
+    adherenceSupportAssigned = models.CharField(max_length=255, null=True)
+
+    @property
+    def current_patient_type_choice(self):
+        if self.newOrRetreatment == 'New':
+            return 'new'
+        elif self.newOrRetreatment == 'Retreatment':
+            if self.retreatmentReason in ['Recurrent', 'Relapse']:
+                return 'recurrent'
+            elif self.retreatmentReason == 'After Treatment Failure':
+                return 'treatment_after_failure'
+            elif self.retreatmentReason == 'After loss to follow-up':
+                return 'treatment_after_lfu'
+            elif self.retreatmentReason == 'Others':
+                return 'other_previously_treated'
+        return ''  # TODO - handle
+
+    @property
+    def disease_classification(self):
+        return {
+            'Pulmonary': 'pulmonary',
+            'Extrapulmonary': 'extra_pulmonary',
+            'Pulmonary+Extrapulmonary': 'extra_pulmonary',
+            'Select': '',
+            'N/A': '',
+        }[self.site]
+
+    @property
+    def hiv_status(self):
+        return {
+            'Positive': 'reactive',
+            'Negative': 'non_reactive',
+            'Not known': 'unknown',
+            'Select': 'unknown',
+            None: 'unknown',
+        }[self.hiv]
+
+    @property
+    def site_choice(self):
```

Comment:
For the keys, I checked what values were in the data dump.

For the values, referenced ```site_choice``` from the public sector: https://india.commcarehq.org/a/enikshay/apps/view/ce3e9160e05acd8e0020ed5e33ca89ea/summary/#/cases

It looks like each list has values missing in the other one - is that expected?